webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -2961,7 +2971,15 @@ var userOverride = {
                     'aria-label',
                     LEAD_COLLECTION_LABEL.submit
                 );
-                leadCollectionHeading.innerHTML = LEAD_COLLECTION_LABEL.heading;
+                if (appOptions.preLeadCollection) {
+                    leadCollectionHeading.innerHTML =
+                        LEAD_COLLECTION_LABEL.heading;
+                } else {
+                    leadCollectionHeading.innerHTML = appOptions.appSettings
+                        .chatWidget.preChatGreetingMsg
+                        ? appOptions.appSettings.chatWidget.preChatGreetingMsg
+                        : " ";
```

Comment:
what is the purpose of space here?

Reply:
by chance, if no value is available in `chatWidget.preChatGreetingMsg` key then no greeting message will be displayed in the pre-chat lead collecton.

Reply:
what will be the difference between "" and " "?

Reply:
ok got it will update it.